### PR TITLE
Calculating totals shipments extracted to helper class Closes #66

### LINKED ISSUE #66: Extract Totals helper module
On [this line](https://github.com/distributeaid/distributeaid.org/blob/saga/src/templates/RegionPage.tsx#L56), we're adding to the page the calculation of a series of shipments' values, as a `totals` object.

It would be good to extract a module that accepts these shipments and returns that values object.

This would allow a consistent way of calculating shipment totals across the codebase.

Previous work done in #26, #65 as a way to resolve #24, see CodeSee recording for latest state:
https://app.codesee.io/shared/c92ab390-3289-11ec-81ae-67b50d769a2b

### PR DESCRIPTION
**Pull request by dolatapatryk**
Refactor for #66

Calculating totals shipments extracted to helper class.

### CODE REVIEW

**Comment by ghost** [comment]
### Changes to your CodeSee Architecture Map:
[View a CodeSee Map of these changes](https://app.codesee.io/r/prs?id=6521&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org)

View [this PR diagram as an interactive map](https://app.codesee.io/r/prs?id=6521&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org)

#### Legend
CodeSee Map Legend

File: .gitignore [hunk 1]
```diff
@@ -70,3 +70,6 @@ public
 
 # Local Netlify folder
 .netlify
+
+#idea
+.idea/
```
Comment (hola-soy-milk) [inline .gitignore]: Is this needed? I think it might be an IDE feature. 

If not, can we remove it?

**Review by hola-soy-milk — CHANGES_REQUESTED** [review]
Thank you so much for your contribution! I left one minor comment in, could you please take a look?

File: .gitignore (hunk 1 above)
Comment (dolatapatryk) [reply .gitignore]: Yep, my IDE generates such directory so I wanted to exclude it, maybe for further purpose too. But I think I can remove this. I will handle it in a minute.

**Comment by dolatapatryk** [comment]
@ramonh Done :)

**Comment by dolatapatryk** [comment]
@ramonh sorry for asking but is my solution good? I'm asking because October is ending and I'm trying to get my hacktober points :)

**Comment by hola-soy-milk** [comment]
Eek! Sorry @dolatapatryk, Dunno why I missed it yesterday. Will take a look right away!

**Review by hola-soy-milk — APPROVED** [review]
Looks brilliant, thank you so much!

**Comment by dolatapatryk** [comment]
@ramonh thank you cery much! 😎

**Pull request merged**